rocky/assets/js/sidemenuOl.js

```diff
@@ -70,3 +74,71 @@ function addToggleButton(sidemenu) {
 
   prependNode(sidemenu, button);
 }
+
+/**
+ * @param {HTMLElement} sidemenu
+ */
+
+function adjustMaxHeight(sidemenu) {
+  let pageHeight = document.documentElement.scrollHeight;
+
+  const pageHeaderElement = document.getElementById("page-header");
+  const pageFooterElement = document.getElementById("page-footer");
+  const stickyElement = document.getElementById("sticky-container");
```

Comment:
Using ID's limits us to only having 1 sidemenu on a page. Could we use hierarchy to select these elements?
Eg, document.getElementById("page-header") could be : document.querySelector('body > header')

Reply:
Good question. I have added this as a ticket as it should be discussed. The choice to use id's was to make it easier to use this component within multiple hierarchy settings (as this is a manon component). But it might be worth to look into this. https://github.com/minvws/nl-kat-coordination/issues/3167

I would suggest to move this pr with the sidemenu fixes forward in the meantime and add any enhancements on this in a separate pr.